src/frequenz/client/microgrid/sensor.py

```diff
@@ -48,3 +73,170 @@ def __str__(self) -> str:
         """Return a human-readable string representation of this instance."""
         name = f":{self.name}" if self.name else ""
         return f"<{type(self).__name__}:{self.id}{name}>"
+
+
+@enum.unique
+class SensorMetric(enum.Enum):
+    """The metrics that can be reported by sensors in the microgrid.
+
+    These metrics correspond to various sensor readings primarily related to
+    environmental conditions and physical measurements.
+    """
+
+    UNSPECIFIED = 0
+    """Default value (this should not be normally used and usually indicates an issue)."""
+
+    TEMPERATURE = 1
```

Comment:
Time to switch to pb2 names instead of numbers now?

Reply:
I'm still not convinced. This is a pure-python class/interface, all the dealing with protobuf conversion is done in the `_from_proto()` functions. I just happen to pick the same numeric IDs for convenience, to make conversion more easy.

What arguments do you have in favor of using the PB values here?

Reply:
> This is a pure-python class/interface

We are trying to build a client for the microgrid API.  Not a stand-alone thing.  This is just increasing the chances for bugs, especially for esoteric component types, which we might add support for only later.  To me, this is just overcomplicating things for no apparent benefit.

> What arguments do you have in favor of using the PB values here?

Single source of truth for enums without any manual work.
and just a few days back, I saw some API PRs changing the numbers for enums.

Reply:
I understand your point about this being very dependant on the defined protobuf stuff, but all wrappers are protobuf free. I know it is a very theoretical thing but I like it. :D

It is a very minor thing, but the alternative to me is also a very minor thing. The single point of truth is pointless once this is stable, is a write once, never ever touch it again, ever. So I'm not concerned at about about it.

That said, since it is controversial, and the API is currently NOT stable, I can see it might be problematic until we get there, and if we have some issue because of this, I don't want you in my face telling me I told you so, so I'll change it and leave the discussion for a distant future :P